### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by Fryuni**

### CODE REVIEW

File: docs/src/content/docs/inline-mod/aik-plugin.md [hunk 1]
```diff
@@ -7,56 +7,63 @@ sidebar:
 
 ## Getting Started
 
-Add the desired plugins (or all plugins) to your `defineIntegration` call:
+Add the plugin to your `withPlugins` call:
 
-```ts ins={2,9-12} ins=/\S(defineMiddlewarePlugin)/ ins=/defineMiddleware(?= )/
-// your-integration.ts
-import { defineMiddlewarePlugin } from '@inox-tools/aik-mod';
+```ts ins={3,13-15} ins=/\S(aikMod)/ ins=/defineMiddleware(?= )/
```
Comment (BryceRussell) [inline docs/src/content/docs/inline-mod/aik-plugin.md:12]:
```suggestion
```ts ins={3,13-16} ins=/\S(aikMod)/ ins=/defineMiddleware(?= )/
```
Highlight is off by one

File: docs/src/content/docs/inline-mod/aik-plugin.md [hunk 2]
```diff
@@ -93,23 +103,26 @@ export default defineIntegration({
 
 `defineMiddleware` allows you to define an Astro middleware inline.
 
-```ts /defineMiddleware\b/ {2,7}
+```ts ins=/defineMiddleware(?= )/ ins={13-17}
```
Comment (BryceRussell) [inline docs/src/content/docs/inline-mod/aik-plugin.md:106]:
```suggestion
```ts ins=/defineMiddleware(?= )/ ins={13-16}
```
Highlight is off by one

File: package.json [hunk 3]
```diff
@@ -27,23 +25,12 @@
...
     "turbo": "^1.13.0",
     "typescript": "^5.4.3"
   },
-  "packageManager": "pnpm@8.15.5+sha256.4b4efa12490e5055d59b9b9fc9438b7d581a6b7af3b5675eb5c5f447cee1a589",
-  "pnpm": {
-    "patchedDependencies": {
-      "@lunariajs/core@0.0.31": "patches/@lunariajs__core@0.0.31.patch"
-    }
-  }
+  "packageManager": "pnpm@8.15.6+sha256.01c01eeb990e379b31ef19c03e9d06a14afa5250b82e81303f88721c99ff2e6f"
```
Comment (florian-lefebvre) [inline package.json:35]: What's the point of using a hash and how do you get it?

File: packages/aik-mod/src/index.ts [hunk 4]
```diff
@@ -50,70 +50,42 @@ function ensurePluginIsInstalled(options: Pick<HookParams, 'config' | 'updateCon
...
-	implementation: ({ config, updateConfig, addMiddleware }) => {
-		const ensurePlugin = ensurePluginIsInstalled({ config, updateConfig });
-		return (order: 'pre' | 'post', handler: MiddlewareHandler) => {
-			ensurePlugin();
-			addMiddleware({
-				order,
-				entrypoint: inlineModule({
-					constExports: {
-						onRequest: handler,
+export default definePlugin({
```
Comment (florian-lefebvre) [inline packages/aik-mod/src/index.ts:53]: Happy to see it simplified stuff drastically!

**Review by florian-lefebvre — APPROVED** [review]
LGTM! I'll just note aik versions are not consistent across packages, being `^0.10.0` or `~0.10`. Just a nitpick tho! I left a few comments about stuff I'm curious about ;)

File: package.json (hunk 3 above)
Comment (florian-lefebvre) [reply package.json:35]: TIL about `corepack use pnpm`!

File: docs/src/content/docs/inline-mod/aik-plugin.md [hunk 5]
```diff
@@ -7,56 +7,63 @@ sidebar:
...
-  setup: () => ({
-    'astro:config:setup': ({ defineMiddleware }) => {
-      defineMiddleware('pre', (context, next) => {
-        // Your inline middleware
-        return next();
-      });
-    },
-  }),
+  name: 'my-integration',
+  setup: ({ name }) => {
```
[the remark this answers is not in the document]
Comment (Fryuni) [reply docs/src/content/docs/inline-mod/aik-plugin.md:19]: For AIK they are the same, but in general they do have a difference.
- `setup() {}` declares a named normal function
- `setup: () => {}` declares an arrow function

It is not two syntaxes for the same thing, they are different objects at runtime with different types. A normal function has access to the object it is called on as `this`. An arrow function doesn't have a receiver parameter, `this` is a normal variable from its scope.

[the remark this answers is not in the document]
Comment (florian-lefebvre) [reply docs/src/content/docs/inline-mod/aik-plugin.md:19]: Alright thx! Yes I was aware of the differences but my question was about aik in particular, so only a matter of pref here thx!

**Pull request merged**